.travis.yml

```diff
@@ -23,6 +23,8 @@ env:
 
 before_install:
   - if test -n "$DATABASE"; then ./scripts/travis-postgresql; fi
+  - docker pull fsfe/reuse:latest
```

Comment:
I think `docker run` already pulls `fsfe/reuse` automagically when it's not locally available.